stablehlo/reference/Element.h

```diff
@@ -41,6 +44,38 @@ class Element {
       : type_(type), value_(std::make_pair(value.real(), value.imag())) {}
 
   Element(const Element &other) = default;
+  Element() = default;
+
+  template <class V>
+  static Element getValue(Type type, V value) {
```

Comment:
Can we use `T` for the type instead? IMO seems more natural for type.

Reply:
I was considering T as well, but it felt weird to have `(Type type, T value)` when we already have a `Type` class as a variable.